src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,90 @@
+import cn from 'classnames';
+import './Dropdown.scss';
+import React, { useState, useMemo, useCallback } from 'react';
+import { Person } from '../../types/Person';
+import { debounce } from '../../services/debounce';
+
+type Props = {
+  peopleList: Person[];
+  onSelected?: (person: Person | null) => void;
+  delay: number;
+};
+
+export const Dropdown: React.FC<Props> = ({
+  peopleList,
+  onSelected = () => {},
+  delay,
+}) => {
+  const [query, setQuery] = useState('');
+  const [visible, setVisible] = useState(false);
```

Comment:
Rename all boolean variables according [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)

```suggestion
  const [isVisible, setIsVisible] = useState(false);
```